src/utils/network/session-core.js

```diff
@@ -112,6 +117,39 @@ export const createPublicContainer = async (session, podUrl) => {
   }
 };
 
+/**
+ * Function that creates a Documents container in the user's Pod when logging in
+ * for the first time or if Documents is missing and initialize it with an ACL with
+ * access to the user
+ *
+ * @memberof utils
+ * @function createDocumentsContainer
+ * @param {Session} session - Solid's Session Object {@link Session}
+ * @param {URL} podUrl - The user's Pod URL
+ * @returns {Promise} Promise - Generates a Documents container for Pod upon log
+ * in if user's Pod does not have the an outbox to begin with
+ */
+export const createDocumentsContainer = async (session, podUrl) => {
```

Comment:
This is fine for now, but I think it's a good idea to break up `session-core` into more specific files with more targeted responsibilities. The name for this is already a bit confusing. `session` and `networking` both imply that these functions are responsible for establishing a connection with the server. But `createDocumentContainer` is clearly devoted to container and file management, which isn't the same as session management.

Reply:
I think I could break this up from session-core, perhaps placing them in a folder named pod_management or something like that.

I could move the other create container functions into there too. I'll give that a go first.